lxd/events.go

```diff
@@ -35,8 +36,14 @@ type eventsServe struct {
 }
 
 // Render starts event socket.
-func (r *eventsServe) Render(w http.ResponseWriter) error {
-	return eventsSocket(r.s, r.req, w)
+func (r *eventsServe) Render(w http.ResponseWriter, request *http.Request) error {
```

Comment:
@hamistao so this commit is a good example of where I would suggesting splitting it up into several commits. 
As when reviewing this commit I am faced with having to scroll to an indeterminate place to find the actual triggering change (this line) amongst all the other usage update changes. I suggest making the change to this function as 1 commit first, and then the subsequent usage updates as the next commit.

Reply:
Ok, thanks for the insight! I guess I overdid my squashing while trying to reduce the size of this PR.